models/comments.models.js

```diff
@@ -12,4 +12,12 @@ exports.insertCommentForArticleId = (article_id, comment) => {
         return db.query(`INSERT INTO comments (article_id, body, author) VALUES ($1,$2,$3) RETURNING *;`, inputArray).then(({ rows }) => {
         return rows[0]
     })
+}
+
+exports.deleteCommentById = (comment_id) => {
+    return db.query(`DELETE FROM comments WHERE comment_id=$1 RETURNING *`, [comment_id]).then(({ rows }) => {
+        if (rows.length === 0) {
+            return Promise.reject({ status: 404, message: "Comment ID not found" })
```

Comment:
won't the rows always be empty? I'm not sure why this works !!!! 🌳 (tree because I'm 'stumped'!)